examples/tls-termination/haproxy/ingress-tls-default.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: extensions/v1beta1
+kind: Ingress
+metadata:
+  name: app
+spec:
+  tls:
+  - hosts:
+    - foo.bar
+  rules:
+  - host: foo.bar
+    http:
+      paths:
+      - path: /
+        backend:
+          serviceName: nginx
```

Comment:
suggest using the "http-svc" deployed in the PREREQUISITES doc if you can: https://github.com/kubernetes/ingress/blob/master/examples/PREREQUISITES.md#test-http-service